app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -485,6 +485,10 @@ public boolean setViewValue(View view, Cursor cursor, int columnIndex) {
                         }
                         return true;
                     case R.id.nameTV:
+                        //currently emojis are in png format, add others if new format(jpg, jpeg etc) is introduced
+                        if (!personTable && name.endsWith(".png")) {
```

Comment:
It would be much better to remove in the database rather than string manipulation! 
https://github.com/zulip/zulip-android/blob/master/app/src/main/java/com/zulip/android/models/Emoji.java